src/data/faq_de.json

```
@@ -193,6 +193,26 @@
                             "<ul><li>In der Regel ist ein in Deutschland ausgestelltes Impfzertifikat ein Jahr ab Ausstellungdatum gültig.</li><li>Ein Genesenenzertifikat darf laut Festlegung der EU dagegen nicht länger als 180 Tage ab dem Datum des letzten positiven PCR-Tests gültig sein.</li></ul>",
                             "Ist Ihr Zertifikat technisch noch gültig, wurde es offenbar von einer nicht autorisierten Stelle oder fehlerhaft ausgestellt. In diesem Fall bitten wir Sie, sich an die <a href='#international_phone_numbers' target='_blank' rel='noopener noreferrer'>technische Hotline</a> zu wenden und uns mitzuteilen, wann und von welcher Stelle dieses Zertifikat ausgestellt wurde. Dann lassen Sie sich bitte ein neues Zertifikat mit gültiger Signatur von einer autorisierten Stelle ausstellen."
                         ]
+                    },
+                    {
+                        "title": "Wie erstelle ich einen Ausdruck meines Digitalen COVID Zertifikates der EU?",
+                        "anchor": "eu_dcc_export",
+                        "active": true,
+                        "textblock": [
+                            "<b>Zweck</b>",
+                            "Ab der Version 2.10 der Corona-Warn-App können Sie eingescannte Digitale COVID Zertifikate der EU in PDF-Dokumente exportieren. Diese Funktion dient vorrangig der Erstellung von Sicherheitskopien der Zertifikate für Ihre persönlichen Zwecke",
+                            "Selbstverständlich werden für das Layout der Zertifikate die Original-EU-Vorlagen benutzt.",
+                            "<b>Sicherheitshinweise</b>",
+                            "Der Export und die Speicherung der Zertifikate sind freiwillig. Das Erstellen der PDF-Datei findet ausschließlich lokal auf Ihrem Smartphone statt. Nur Sie selbst haben Zugriff auf dieses Dokument.",
+                            "Beachten Sie aber, dass das erzeugte Dokument sensible persönliche Informationen enthält. Gehen Sie daher bitte sorgsam mit der PDF-Datei um. Zeigen Sie das Dokument nur solchen Personen vor, denen Sie vertrauen und die zur Prüfung auch berechtigt sind.",
+                            "Wir raten dringend davon ab, das PDF-Dokument zu veröffentlichen über potentiell unsichere Kanäle (z.B. per E-Mail oder über andere Apps) zu teilen.",

```

```suggestion
                            "Wir raten dringend davon ab, das PDF-Dokument zu veröffentlichen über potentiell unsichere Kanäle (z.&nbsp;B. per E-Mail oder über andere Apps) zu teilen.",
```
See https://github.com/corona-warn-app/cwa-website/issues/1689 regarding [DIN 5008](https://de.wikipedia.org/wiki/DIN_5008#Abk%C3%BCrzungen_(Abschnitt_4.5))